tests/dragonfly/instance.py

```diff
@@ -9,6 +9,7 @@
 import itertools
 from prometheus_client.parser import text_string_to_metric_families
 from redis.asyncio import Redis as RedisClient
+import signal
```

Comment:
how come you needed to add this import now?

Reply:
I had randomly had a deadlock when I run the tests locally, and it couldn't sent a signal so I imported it

Reply:
so does this means this is the first time that this flow is executed? I so, please make sure this flow works well by making a deadlock in dfly , just add stupid code that creates a deadlock and check the pytests works as expected

Reply:
No not really. I tested this heavily but somehow I removed it while polishing the PR (I was running it on the CI so probably while I reverted the changes I also removed the import...)

Reply:
ok